scriptworker/test/test_config.py

```diff
@@ -56,12 +56,23 @@ def test_freeze_values(input_dict, expected_dict):
     assert input_dict == expected_dict
 
 
-@pytest.mark.parametrize("input_dict,expected_dict", (({1: 2}, {1: 2}), ({1: (1, 2)}, {1: [1, 2]}), ({1: frozendict({2: 3})}, {1: {2: 3}}),
-                                                      ({1: (frozendict({2: 3}), (4, 5, 6))}, {1: [{2: 3}, [4, 5, 6]]}),
-                                                      ({1: (1, (2, 3), frozendict({4: 5}))}, {1: [1, [2, 3], {4: 5}]}), ((1, 2), (1, 2))))
+@pytest.mark.parametrize("input_dict,expected_dict", ((
+    {1: 2}, {1: 2}
```

Comment:
This shouldn't matter anymore with the list/dict comprehension version, but we can make the data types more varied. For instance:
```py
@pytest.mark.parametrize("input_values, expected", ((
    {1: 2}, {1: 2}
), (
    {'a': (1.0, 2.5)}, {'a': [1.0, 2.5]}
), (
    {1: frozendict({'a': True})}, {1: {'a': True}}
), (
    {1: (frozendict({'a': None}), ('A', 'B', 'C'))}, {1: [{'a': None}, ['A', 'B', 'C']]}
), (
    {1: (1, [2, 3], frozendict({}))}, {1: [1, [2, 3], {}]}
), (
    (1, ()), [1, []]
), (
    1, 1
)))
def test_unfreeze_values(input_values, expected):
    assert config.unfreeze_values(input_values) == expected
```